rust/routee-compass-core/src/model/cost/cost_configuration.rs

```diff
@@ -0,0 +1,9 @@
+use super::vehicle::vehicle_cost_mapping::VehicleCostMapping;
+
+pub struct CostConfiguration {
```

Comment:
Looks like this might be an artifact of a previous iteration as I can't find it linked in anywhere